.travis.yml

```diff
@@ -95,13 +95,11 @@ jobs:
         - yarn test
 
     - name: Node.js Tests
-      node_js: "8"
```

Comment:
The default `node_js` is specified as 10 above, removing this line makes it use 10 instead of 8.